framework/helpers/BaseHtml.php

```diff
@@ -434,7 +434,18 @@ public static function mailto($text, $email = null, $options = [])
      */
     public static function img($src, $options = [])
     {
-        $options['src'] = Url::to($src);
+        if (isset($src)) {
+            $options['src'] = Url::to($src);
+        }
```

Comment:
@rob006 that is [true](https://www.w3.org/TR/html5/embedded-content-0.html#the-img-element).

> The src attribute must be present, and must contain a valid non-empty URL potentially surrounded by spaces referencing a non-interactive, optionally animated, image resource that is neither paged nor scripted.